packages/keystone/src/admin-ui/templates/index.ts

```diff
@@ -41,7 +46,7 @@ export const writeAdminFiles = (
         {
           mode: 'write' as const,
           src: apiTemplate,
-          outputPath: 'pages/api/graphql.js',
+          outputPath: `pages/${config.graphql?.path || '/api/path'}.js`,
```

Comment:
In addition to the change below, this won't work if the path isn't within `/api` so we should either throw an error if `enableNextJsGraphqlApiEndpoint` is enabled and the path isn't within `/api` or remove `enableNextJsGraphqlApiEndpoint`.

```suggestion
          outputPath: `pages/${config.graphql?.path || '/api/graphql'}.js`,
```